env.default

```diff
@@ -33,12 +33,6 @@ XROBOTSTAG_ENABLED=False
 # See the Basket client docs https://basket-client.readthedocs.io/en/latest/install.html
 BASKET_URL=https://basket-dev.allizom.org/
 
-# Newsletter config:
```

Comment:
Added in [#6552](https://github.com/mozilla/foundation.mozilla.org/pull/6552/files#diff-14d0b30702e03f0c4431c6bb68320bc3b0d1660ab34d9e501d28b503d424d42aR34) - no longer needed